### ISSUE TITLE
[joomla.org] Red alert, shields up - The work of the Joomla Security Team

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-websites#1232]
This is in the wrong category. It has been placed in Release news which is the category used by joomla cms in the post-installation messages to display releases. Nothing else should be there.

**Comment by Sandra97** [comment]
You're totally right. Thanks for having noticed. It's now in the "general news" category.
